### PR TITLE
Remove references to nonexistent logger, add logging to sending and receiving frames

### PR DESCRIPTION
**Pull request by peterwilson**

### CODE REVIEW

**Comment by leekirchhoff** [comment]
looks good

File: lib/amqp/session.rb [hunk 1]
```diff
@@ -130,11 +130,6 @@ def logger
       end
 
       def logger=(logger)
```
Comment (sherriman) [inline lib/amqp/session.rb:132]: `def logger` is no longer needed because it is defined as an attor_accesor.

Comment (peterwilson) [inline lib/amqp/session.rb:132]: It's set as an attr_accessor for the instance, and these methods are defined at the class level.

Comment (sherriman) [inline lib/amqp/session.rb:132]: Well kill me, I can't see that because of collapsed lines :)

**Comment by sherriman** [comment]
:+1:

**Pull request merged**